FrontEnd/myapp/src/Stock.js

```diff
@@ -9,11 +13,20 @@ class Stock extends React.Component {
     this.state = {
       stockChartXValues: [],
       stockChartYValues: [],
+      ticker: this.props.symbol,
+      articles: [],
+      apiError: "",
     };
   }
 
-  componentDidMount() {
+  async componentDidMount() {
     this.fetchStock();
+    try {
+      const response = await articles(this.state.ticker);
+      this.setState({ articles: response.articles });
+    } catch (error) {
+      this.setState({ apiError: "Could not find anything" });
+    }
```

Comment:
This is important for error checking, good change.

Reply:
I think we need to add more error checking elsewhere in the function